src/Search/Api/Query/RamlRequirement.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace eLife\Search\Api\Query;
+
+trait RamlRequirement
+{
+    public static $root = __DIR__.'/../../../../vendor/elife/api/dist/samples/';
```

Comment:
The relevant Puli mapping is https://github.com/elifesciences/api-raml/blob/develop/puli.json#L4-L7. But in this case I think it's a false abstraction, and the fixtures should live here (sounds like the test could start to fail if a legitimate change is made in the other repo).